### DISCUSSION TITLE
OpenSSL, Bouncy Castle and Cryptlib

### DISCUSSION THREAD

**Discussion by beldmit** [openssl/openssl#24997] [General Discussion]
I would like to better understand the relationship between OpenSSL, Bouncy Castle and Cryptlib. Is the alliance only a mission-level agreement, or there will be some other more practical steps (common implementations, common CI env, alignment of the policy etc)?

**Comment by mattcaswell** [comment]
At this stage the agreement is that the projects adhere to the OpenSSL mission and values only. By agreeing to do so this enables the Foundation and Corporation to offer support to the projects on a purely opt-in voluntary basis. We hope to explore ways that we can collaborate and build on this initial agreement which may lead to more practical steps at some future time.
